float64 motor_current
float64 motor_rpm
uint16 motor_pulses
float64 motor_temperature

float64 driver_voltage
float64 driver_temperature
uint8 driver_state